ansible_base/authentication/serializers/authenticator_map.py

```diff
@@ -27,14 +24,30 @@ def validate(self, data) -> dict:
         if map_type == 'organization' and (not org or org == ''):
             errors["organization"] = "You must specify an organization with the selected map type"
 
-        if not data.get('order', None):
-            errors['order'] = "Must be a valid integer"
-
         if errors:
             raise ValidationError(errors)
         return data
 
-    def validate_trigger_data(self, triggers: dict, definition, error_prefix: str) -> dict:
+    def validate_trigger_data(self, data):
+        errors = {}
+        request = self.context.get('request', None)
+        if 'triggers' not in data or not data['triggers']:
+            if not request or (request.method != 'PATCH'):
+                errors["triggers"] = "Triggers must be a valid dict"
```

Comment:
Final drive-by comment about this

https://github.com/ansible/django-ansible-base/blob/5babe8742e57d45ea703986398c2777fb370f991/ansible_base/authentication/models/authenticator_map.py#L64-L69

If this shouldn't be left out of create data, or set to `{}` (the default) by omitting from PUT data, then what you're really asking for starts to sound like `blank=False`.

Reply:
that's true, the question is, if it was the original intention that empty dict is not possible. 

@john-westcott-iv do you know more about this?

Reply:
@AlanCoding ... the intention is not to allow `{}`. I tried `blank=False`, which doesn't work for this condition, it's not equal to this validation. So it has to stay here